src/evolutions/src/main/scala/play/api/db/slick/evolutions/internal/DBApiAdapter.scala

```diff
@@ -49,18 +49,21 @@ private[evolutions] object DBApiAdapter {
       }
     }
     def url: String = dbConfig.db.createSession().metaData.getURL
-    def getConnection(): Connection = {
+    def getConnection(): Connection = getConnection(autocommit = true)
```

Comment:
Actually, I'd prefer to keep the implementation of `getConnection()` as it was, and only change `getConnection(autocommit: Boolean)` as follow:

```
def getConnection(autocommit: Boolean): Connection = {
  val conn = getConnection()
  conn.setAutoCommit(autocommit)
  conn
}
```

Same applies to `withConnection`.